handler/proxy.go

```diff
@@ -319,6 +324,19 @@ func (p *Proxy) Director(req *http.Request) error {
 		if p := getAttribute(evalContext, "path", content); p != "" {
 			path = p
 		}
+		if del, ok := content.Attributes["remove_query_params"]; ok {
```

Comment:
"Collecting" and setting the query params should be extracted to a method.

The director method is actually a setup method to prepare the correct dial parameters.

Please refer to `SetRoundtripContext` to make this kind of modifications.